src/navigation/ModalStack.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from "react"
+import { createStackNavigator } from "@react-navigation/stack"
+
+import { Stacks, ModalScreens } from "./index"
+import LanguageSelection from "../Modal/LanguageSelection"
```

Comment:
What do you think about not having a `Modal` folder?  (and if you decide to keep it, what do you think about `modal/` as convention is CamelCase for specific components, and `Modal` here is not a specific component.)

Reply:
Ah I see, using `modal` makes sense to me if that's cool.